src/archetype/ArchetypeModule.ts

```diff
@@ -55,7 +55,13 @@ export namespace ArchetypeModule {
         sendInfo(operationId, { archetypeArtifactId: artifactId, archetypeGroupId: groupId });
 
         // choose target folder.
-        const cwd: string = await instrumentOperationStep(operationId, "chooseTargetFolder", chooseTargetFolder)(entry);
+        let targetFolderHint: Uri;
+        if (entry) {
+            targetFolderHint = entry;
+        } else if (workspace.workspaceFolders.length > 0) {
+            targetFolderHint = workspace.workspaceFolders[0].uri;
```

Comment:
Do we need to take the multi-root into consideration?

Reply:
How to? I can think of two ways: 1) ask users to select a root. 2) infer from active file.

1) introduces an extra step. 2) doesn't make much sense and you need to handle case where no active file is open. Meanwhile, it's possible that user want to create a project anywhere else. It's just a default location for the file selector. 

My point is, it's better than previous behavior without adding extra steps. So I prefer not to support that now.